### PR TITLE
Adding Event Hub namespace type to Service Bus ServiceManagement

### PR DESCRIPTION
**Pull request by dsouzaarun**
Adding support for 'EventHub' namespace type to Service Bus ServiceManagement API

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@dsouzaarun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ardsouza). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@dsouzaarun No tests?

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@dsouzaarun Please add a test that exercises the new enum value.

File: src/ServiceManagement/ServiceBus/ServiceBus.Tests/ClientTest.cs [hunk 1]
```diff
@@ -73,6 +70,49 @@ public void CanCreateNamespace()
...
+
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                Assert.Equal(namespaceName, response.Namespace.Name);
+                Assert.Equal(location, response.Namespace.Region);
+                Assert.Equal(false, response.Namespace.CreateACSNamespace);
+                Assert.Equal(NamespaceType.EventHub, response.Namespace.NamespaceType);
+                int retries = 0;
+                while (true)
+                {
+                    Thread.Sleep(TimeSpan.FromSeconds(4));
```
Comment (hovsepm) [inline src/ServiceManagement/ServiceBus/ServiceBus.Tests/ClientTest.cs]: Please wrap this with the if statement where you check that the current mode is not a PlayBack. just add
if (HttpMockServer.Mode != HttpRecorderMode.Playback)
{

**Pull request merged**